packages/valory/skills/abstract_round_abci/test_tools/base.py

```diff
@@ -415,7 +415,8 @@ def _test_done_flag_set(self) -> None:
     @classmethod
     def teardown_class(cls) -> None:
         """Teardown the test class."""
-        _MetaPayload.registry = cls.old_tx_type_to_payload_cls
+        if getattr(cls, "old_tx_type_to_payload_cls", False):
```

Comment:
Where is this used?

Reply:
It's because of the `test_setup_fails_without_path` (91719607ea59a9be5a8184e1ff0dad8a245b605c).